### PR TITLE
Multiple aws profiles

### PR DESCRIPTION
**Pull request by PrestigeTicko**
#### Card

Implemented support for multiple AWS profiles via a text file of multiple profiles separated by new lines or simply asking cloudfox to use all profiles in the credentials file. 

#### Details

* The init function of cli/aws.go has a call to a new function called initAWSProfiles which sets values to a slice of strings which is used as the AWS profiles to be used for the remaining commands. 
* The initAWSProfiles function at cli/aws.go:855 ensures that only one option of list of profiles, one profile, or all profiles is selected. It then uses the new util functions to get the AWS profiles from a given file or the default AWS credentials file. As noted in the readme of cloudfox, this functionality also assumes the user will have an AWS credentials file in place. When neither of -p, --profiles-list (new), or --all-profiles (new) is selected and empty string is passed and existing functionality takes care of using the AWS environment variables or the default profile.
*  The two helper functions that are used in initAWSProfiles are utils.GetSelectedAWSProfiles and utils.GetAllAWSProfiles. Both do what their names suggest and I have included tests in utils/aws_test.go.
* To accommodate multiple profiles I simply wrapped a for loop iterating over the list of profiles around every function call. Thats it :)
* The --all-profiles / -a and --profiles-list / -l flags were added.
* Some modules were updated in the process and I also updated a small logging statement in AWSConfigFileLoader in utils/aws.go so that it would specify the given profile its having trouble reading.

### CODE REVIEW

**Review by B-Eazy6680 — CHANGES_REQUESTED** [review]
EDITED: removing content because of SPAM.

File: go.mod [hunk 1]
```diff
@@ -10,15 +10,15 @@ require (
 	github.com/Azure/go-autorest/autorest/to v0.4.0 // indirect
 	github.com/Azure/go-autorest/autorest/validation v0.3.1 // indirect
 	github.com/aquasecurity/table v1.8.0
-	github.com/aws/aws-sdk-go-v2 v1.16.15
+	github.com/aws/aws-sdk-go-v2 v1.16.16
 	github.com/aws/aws-sdk-go-v2/config v1.13.0
```
[the remark this answers is not in the document]
Comment (B-Eazy6680) [reply go.mod:14]: Why

File: go.mod [hunk 2]
```diff
@@ -10,15 +10,15 @@ require (
...
 	github.com/aws/aws-sdk-go-v2/service/apigateway v1.15.6
```
Comment (B-Eazy6680) [inline go.mod:15]: **[]()[]()[]()[]()**

**Review by B-Eazy6680 — COMMENTED** [review]
- [ ]

**Comment by sethsec-bf** [comment]
- For `--all-profiles`, added check for profiles in config (in addition to credentials)
- For `--all-profiles`, added a default interactive sanity check to display all profiles and confirm you wan't to really do it
- For `--all-profiles`, added a `-y` flag for non-interactive mode so you can skip the interactive prompt if running in a script
- Updated error messages to include which profile failed
- Added a way for cloudfox to continue in multi-profile mode if one of the many profiles fails

**Review by sethsec-bf — APPROVED** [review]

**Pull request merged**